docs/howto/configure-global-pull-secrets.md

```diff
@@ -0,0 +1,53 @@
+---
+tags:
+  - alpha
+---
+
+# Configure global pull secrets for allowing components to pull private images
+
+**Note: The UX for how auth info for using private images is provided is an active work in progress.**  
+
+To configure `catalogd` and `operator-controller` to use authentication information for pulling private images (catalog/bundle images etc), the components can be informed about a kubernetes `Secret` object that contains the relevant auth information. The `Secret` must be of type `kubernetes.io/dockerconfigjson`. 
```

Comment:
```suggestion
To allow [catalogd](https://github.com/operator-framework/catalogd) and [operator-controller](https://github.com/operator-framework/operator-controller) to authenticate and pull images from a private registry (for catalog/bundle images, etc.), you need to provide these components with a Kubernetes `Secret` object containing the necessary credentials. The `Secret` must be of type [`kubernetes.io/dockerconfigjson`](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials).
```

- The correct term would be "private registry," right, not "images"?
- Additionally, wdyt about adding links to both project repositories for catalogd and operator-controller?
- wdyt about including a reference to the Kubernetes documentation on kubernetes.io/dockerconfigjson secrets for further guidance?